cgroup_top.py

```diff
@@ -49,6 +50,38 @@ def cmd_exists(cmd):
     except OSError:
         return False
 
+
+def docker_container_name(container_id, default, cache=dict()):
+    cached_name = cache.get(container_id)
+    if cached_name:
+        return cached_name
+
+    sp = subprocess.Popen(['docker', 'inspect', container_id], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
If ``docker`` binary is not in the PATH, it will raise ``OSError: [Errno 2] No such file or directory``